src/components/pages/PointShare/RenderPointShare.js

```diff
@@ -51,10 +66,49 @@ const PointShare = props => {
   };
 
   useEffect(() => {
+    const setVirtualPlayerPoints = virtualPlayerID => {
```

Comment:
Can this function and it's specification be moved alongside of the virtualPlayIds  (outside of the useEffect) ?
I think it may improve readability.